mappings/net/minecraft/client/render/ParticleManager.mapping

```diff
@@ -1,9 +1,26 @@
 CLASS net/minecraft/class_255 net/minecraft/client/render/ParticleManager
 	FIELD field_1024 world Lnet/minecraft/class_101;
 	FIELD field_1026 textureManager Lnet/minecraft/class_198;
+	FIELD field_1027 random Ljava/util/Random;
 	METHOD <init> (Lnet/minecraft/class_101;Lnet/minecraft/class_198;)V
 		ARG 1 world
 		ARG 2 textureManagaer
+	METHOD method_644 addBlockBreakingParticles (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_645 addBlockBreakingParticles (IIII)V
+		COMMENT direction:
+		COMMENT \tDOWN: \t0
+		COMMENT \tUP:\t\t1
+		COMMENT \tNORTH:\t2
+		COMMENT \tSOUTH:\t3
+		COMMENT \tWEST: \t4
+		COMMENT \tEAST:\t5
```

Comment:
Inconsistent spaces / tabs after the comments...?
Would it work to instead use a table? Javadoc does support (some) HTML tags

Reply:
I should just use a table. When I looked at it in Enigma, it was not lined up in the normal editor if it was lined up in the javadoc editor